src/css/components/_progress.scss

```diff
@@ -15,6 +15,7 @@
 .video-js .vjs-progress-control {
   @include flex(auto);
   @include display-flex(center);
+  margin: 0 0.45em 0 0.45em;
```

Comment:
Hmm, I am not sure how we would go about fixing this issue if the default styles directly conflict with other skins.

The default is embedded in the control bar, but the other is not...? I guess its kind of chicken/egg'ish.

If it makes sense for the default implementation to be the primary one, then the other skins would need to undo these changes?

The twitchy skin implements https://github.com/mmcc/videojs-skin-twitchy/blob/master/src/components/_slider.scss#L6 (which removes the inline `SeekBar` margin). It would also need to remove the `.vjs-progress-control` ones too (if they were added).

---

Not saying that this should be the case, just that the styles are conflicting—And its weird for the default implementation to have a `MouseTimeDisplay` that goes out of bounds, especially on the home page :p.